src/pages/128x64x1/timer_page.c

```diff
@@ -59,21 +59,21 @@ static int row_cb(int absrow, int relrow, int y, void *data)
     y += space;
     /*prem-timer reset */
     GUI_CreateLabelBox(&gui->resetpermlbl, LABEL_X, y,
-            LABEL_WIDTH, LINE_HEIGHT, &LABEL_FONT, NULL, NULL, _tr("Reset"));
+            LABEL_WIDTH, LINE_HEIGHT, &LABEL_FONT, GUI_Localize, NULL, _tr_noop("Reset"));
     GUI_CreateButtonPlateText(&gui->resetperm, RESET_X, y,
             RESET_WIDTH, LINE_HEIGHT, &BUTTON_FONT, show_timerperm_cb, reset_timerperm_cb,(void *)(long)absrow);
     if(Model.mixer_mode != MIXER_STANDARD) {
         /* or Reset switch */
     	GUI_CreateLabelBox(&gui->resetlbl, LABEL_X, y,
-            LABEL_WIDTH, LINE_HEIGHT, &LABEL_FONT, NULL, NULL, _tr("Reset sw"));
+            LABEL_WIDTH, LINE_HEIGHT, &LABEL_FONT, GUI_Localize, NULL, _tr_noop("Reset sw"));
     	GUI_CreateTextSourcePlate(&gui->resetsrc, TEXTSEL_X, y ,
```

Comment:
Lint Error:
```
64:  Missing space after ,  
64:  Use int16/int64/etc, rather than the C type long  
65:  Missing space before ( in if(  
67:  Tab found; better to use spaces  
69:  Tab found; better to use spaces  
```